core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -93,6 +109,14 @@ export class ResultPanelComponent {
         this.resultPanelToggleService.openResultPanel();
       }
     });
+
+    // clear session storage for refresh
+    sessionStorage.removeItem('newWorkflowExecuted');
+    sessionStorage.removeItem('currentResult');
+    sessionStorage.removeItem('currentPageIndex');
+    sessionStorage.removeItem('currentPageSize');
+    sessionStorage.removeItem('total');
+    sessionStorage.removeItem('columnKeys');
```

Comment:
move session storage operations to separate functions, also maybe we can just put all of these into a single json object and use a single key to save it? Also make the key specific, something like "texera-result-panel-xxx"

Reply:
Updated in commit https://github.com/Texera/texera/pull/903/commits/d668ef2d6eb6c5ef694c00eeaa984ccf51bd860e